requirements.txt

```diff
@@ -15,5 +15,10 @@ mozrunner==6.9
 moztest==0.7
 mozversion==1.4
 
+marionette-client == 2.0.0
+marionette-driver == 1.1.1
+
 # Also pin external packages to minimize the risk of version conflicts
 requests==2.7.0
+
+.
```

Comment:
It's the current folder which is used to install the firefox ui tests package. I will add a comment.